.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -12,11 +12,11 @@ jobs:
     matrix:
       win_c_compilervs2015cxx_compilervs2015python3.6:
         CONFIG: win_c_compilervs2015cxx_compilervs2015python3.6
-        CONDA_BLD_PATH: D:\\bld\\
+        CONDA_BLD_PATH: C:\\bld\\
```

Comment:
We need to either make that `default` or make it an option in `conda-forge.yml`, reverting that at every re-render is silly.